docs/reference/db/introduction.md

```diff
@@ -42,10 +42,10 @@ Get up and running in 2 minutes using our
 
 ## Supported databases
 
-- [SQLite](https://www.sqlite.org/)
-- [PostgreSQL](https://www.postgresql.org/)
-- [MySQL](https://www.mysql.com/)
-- [MariaDB](https://mariadb.org/)
+- [SQLite](https://www.sqlite.org/) >= 5.1.4
```

Comment:
Where does this number come from? As of today the latest version of SQLite is [Version 3.41.0](https://www.sqlite.org/releaselog/3_41_0.html). I think we can safely say it's v3.

Reply:
Sure, changing it.